docs/getting-started.md

```diff
@@ -0,0 +1,58 @@
+# Getting Started
```

Comment:
Would be good to clarify what this is getting started *for*

Maybe something like:

```suggestion
# Getting Started: Building your own addoon

To get started using bpy-build to manage building and reloading your own blender addon project, follow the steps below.
```